android/app/src/main/AndroidManifest.xml

```diff
@@ -21,13 +21,8 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
     <!-- these permissions should be removed -->
-    <!-- react-native-orientation adds an unnecessary permission; here we remove it -->
-    <uses-permission android:name="android.permission.BLUETOOTH" tools:node="remove"/>
-    <!-- react-native-contacts -->
-    <uses-permission android:name="android.permission.WRITE_CONTACTS" tools:node="remove"/>
     <!-- react-native-camera -->
     <uses-permission android:name="android.permission.RECORD_AUDIO" tools:node="remove"/>
-    <uses-permission android:name="android.permission.RECORD_VIDEO" tools:node="remove"/>
     <!-- React Native unnecessary permissions (https://github.com/facebook/react-native/issues/5886) -->
     <uses-permission android:name="android.permission.READ_PHONE_STATE" tools:node="remove"/>
```

Comment:
This one is still necessary?

Reply:
Yes, looks like it's pulled in by instabug dependency:
```
~/dev/status-react/node_modules $ grep -r RECORD_AUDIO  * | grep AndroidManifest
instabug-reactnative/android/build/intermediates/exploded-aar/com.instabug.library/instabug/3.4.0/AndroidManifest.xml:    <uses-permission android:name="android.permission.RECORD_AUDIO" />
~/dev/status-react/node_modules $ 
```

Reply:
Could you update the comment to add this info?